### ISSUE TITLE
x11-misc/i3blocks-9999 replace ruby-ronn with pandoc

### ISSUE THREAD

**Issue by soredake** [anders-larsson/gentoo-overlay#18]
https://github.com/vivien/i3blocks/commit/b736a2732cfea7c4869ae34c6d2a900471a9348e

**Comment by anders-larsson** [comment]
Hi,

Yeah I was actually working already working on it. It has been fixed by https://github.com/anders-larsson/gentoo-overlay/commit/c914acf2e4938dc8253257af9542d4f1119bd3f8 and https://github.com/anders-larsson/gentoo-overlay/commit/fdad33aaa70f319cd69fc62959fd2f665638f980. Before https://github.com/anders-larsson/gentoo-overlay/commit/fdad33aaa70f319cd69fc62959fd2f665638f980 the dependency for building man pages was never used. I've added a doc USE flag to decide if pandoc should be included or it. When I tested it, it required me to merge 90 additional packages.
